compiler/ballerina-lang/src/main/java/org/wso2/ballerinalang/util/TomlParserUtils.java

```diff
@@ -0,0 +1,46 @@
+package org.wso2.ballerinalang.util;
```

Comment:
Missing license header

Reply:
Easier if intelliJ is configured for copyright header.

Reply:
Added missing license headers with 3fa8e1e

Reply:
Year should be 2018.  Shall we fix the IDE first ?

Reply:
Fixed the year to be 2018 with 68e3afc